common/presenters/move-to-meta-list-component.js

```diff
@@ -4,18 +4,26 @@ const moveAgreedField = require('../../app/move/app/new/fields/move-agreed')
 const componentService = require('../../common/services/component')
 const i18n = require('../../config/i18n')
 const filters = require('../../config/nunjucks/filters')
-const getUpdateLinks = require('../helpers/move/get-update-links')
+const mapUpdateLink = require('../helpers/move/map-update-link')
 
+/**
+ * Convert a move into the structure required to render
```

Comment:
I've moved this method to accept an entity as the first argument, which will be required, and then to be able to pass options in as an Object as the second argument. That way, if we ever need to extend/reduce options it will have less of an impact on existing implementations that might only want to pass a single option in. How does this approach sound?